api/PclusterApiHandler.py

```diff
@@ -673,13 +674,30 @@ def login():
 
 
 def logout():
+    refresh_token = request.cookies.get('refreshToken', None)
+    if refresh_token is not None:
+        revoke_cognito_refresh_token(refresh_token)
```

Comment:
question: is this blocking the response? meaning, will the client (FE) need to wait on the new POST request, before getting a response?

Reply:
Yes unfortunately
I didn't make it async because running in a lambda, we would still need to wait for the call to complete or we may interrupt the request when lambda "turns down"